app/views/shared/_postcode-lookup.html.erb

```diff
@@ -12,11 +12,12 @@
           label: {
             text: input_label
           },
+          heading_level: 2,
           heading_size: "s",
           error_message: input_error,
           value: input_value,
           name: "postcode-lookup",
-          hint: hint_text,
+          hint: sanitize(t("coronavirus_local_restrictions.lookup.hint_text")),
```

Comment:
Is there a reason why we don't use the variable above https://github.com/alphagov/collections/pull/2026/files#diff-cd642c40804c0a24c6e913fda95a4ba7cd90b1918f2585c36749622ec36f59b2R5

Reply:
```suggestion
          hint: hint_text,
```

Reply:
Change line 5 to: ```  hint_text ||= sanitize(t("coronavirus_local_restrictions.lookup.hint_text"))```